docs/climate.md

```diff
@@ -51,6 +51,7 @@ Climate are representations of KNX HVAC/Climate controls.
 - `operation_modes` Overrides the supported operation modes.
 - `controller_modes` Overrides the supported controller modes.
 - `device_updated_cb` Callback for each update.
+- `group_address_humidity` KNX address of current room humidity. *DPT 9.007*
```

Comment:
Can you move this up to ~after `group_address_command_value_state` to not get it mixed up with the `ClimateMode` attributes? (there should probably be an extra block for those 😬)